File: jvm/core/src/main/java/se/idkollen/client/models/BankIdSeVerifyResponse.java
package se.idkollen.client.models;

import org.jspecify.annotations.Nullable;

/** Response from the BankID SE QR code verification endpoint. */
public class BankIdSeVerifyResponse {
    public String ssn = "";
    public String name = "";
    public String givenName = "";
    public String surname = "";
    public @Nullable Integer age;
    public @Nullable String verifiedAt;
}

File: php/src/Models/BankIdSe/BankIdSeVerifyResponse.php
<?php

declare(strict_types=1);

namespace Idkollen\Client\Models\BankIdSe;

readonly class BankIdSeVerifyResponse
{
    public function __construct(
        public string $ssn,
        public string $name,
        public string $givenName,
        public string $surname,
        public ?int $age,
        public ?string $verifiedAt,
    ) {}
}
